### PR TITLE
test: add test proving limit/offset for bigint is fixed

### LINKED ISSUE #7212: Wrong result set produced by JDBC driver when passing LIMIT and OFFSET as Long bind values
### What happens?

When passing both `LIMIT` and `OFFSET` as `Long` bind values, it seems the values are ignored

### To Reproduce

```java
try (Statement s = connection.createStatement()) {
    try {
        s.executeUpdate("create table t (i int not null)");
        s.executeUpdate("insert into t values (1), (1), (2), (3), (3), (3)");

        try (PreparedStatement ps = connection.prepareStatement("select t.i from t order by t.i limit ? offset ?")) {
            ps.setLong(1, 2);
            ps.setLong(2, 1);

            try (ResultSet rs = ps.executeQuery()) {
                while (rs.next()) {
                    System.out.println(rs.getInt(1));
                }
            }
        }
    }
    finally {
        s.executeUpdate("drop table t");
    }
}
```

This produces:

```
1
1
2
3
3
3
```

When it should produce

```
1
2
```

The workaround is to pass the `OFFSET` value  as `int`:

```java
ps.setInt(2, 1);
```

### OS:

Microsoft Windows [Version 10.0.22621.1555]

### DuckDB Version:

v0.7.2-dev2430

### DuckDB Client:

JDBC

### Full Name:

Lukas Eder

### Affiliation:

Data Geekery / jOOQ

### Have you tried this on the latest `master` branch?

- [X] I agree

### Have you tried the steps to reproduce? Do they include all relevant data and configuration? Does the issue you report still appear there?

- [X] I agree

### PR DESCRIPTION
**Pull request by Mause**
Fixes https://github.com/duckdb/duckdb/issues/7212

This was actually fixed in https://github.com/duckdb/duckdb/pull/8519, this is just to prove it also fixes this case

### CODE REVIEW

File: tools/jdbc/src/main/java/org/duckdb/DuckDBNative.java [hunk 1]
```diff
@@ -46,13 +47,15 @@ public class DuckDBNative {
             Path lib_file = Files.createTempFile("libduckdb_java", ".so");
             URL lib_res = DuckDBNative.class.getResource(lib_res_name);
             if (lib_res == null) {
-                throw new IOException(lib_res_name + " not found");
+                System.load(
+                    Paths.get("../../build/debug/tools/jdbc", lib_res_name).normalize().toAbsolutePath().toString());
```
Comment (Mytherin) [inline tools/jdbc/src/main/java/org/duckdb/DuckDBNative.java:51]: Should this maybe get an `#ifdef DEBUG` or similar? I suppose we don't want this code in the distributed jars? Perhaps it doesn't matter though.

Comment (Mytherin) [reply tools/jdbc/src/main/java/org/duckdb/DuckDBNative.java:51]: Do you think adding that makes sense or not? If not I'm fine with leaving it as it is.

Comment (Mause) [reply tools/jdbc/src/main/java/org/duckdb/DuckDBNative.java:51]: I think the likelihood of it running is pretty tiny - we can revisit it if we ever go down the path of distributing the native libraries outside the default jar

**Comment by Mytherin** [comment]
Thanks!

**Pull request merged**